.gitignore

```diff
@@ -19,6 +19,7 @@ pip-selfcheck.json
 log
 portal/test_uploads
 docker/portal.env
+docker/docker-compose.pb.yaml
```

Comment:
```suggestion
docker/docker-compose.*.yaml
```

Reply:
Hmm, but that would also hide `docker-compose.{dev,flower,prod}.yaml` - should we try `docker-compose.personal.yaml` or a different pattern, such as `docker-compose.dev.*.yaml`?

Reply:
>Hmm, but that would also hide docker-compose.{dev,flower,prod}.yaml

You would have to use `--force` with `git add` on those files, but I don't think any other change

>should we try docker-compose.personal.yaml or a different pattern, such as docker-compose.dev.*.yaml

Would prefer a wildcard to allow for some flexibility, the second pattern (`docker-compose.dev.*.yaml`) works for me though